src/robusta/core/discovery/discovery.py

```diff
@@ -254,6 +256,39 @@ def discovery_process() -> DiscoveryResults:
             )
             raise e
 
+        # discover helm state
+        helm_releases_map: dict[str, HelmRelease] = {}
+        try:
+            continue_ref: Optional[str] = None
+            for _ in range(DISCOVERY_MAX_BATCHES):
+                secrets = client.CoreV1Api().list_secret_for_all_namespaces(label_selector=f"owner=helm",
+                                                                            _continue=continue_ref)
+                if not secrets.items:
+                    logging.debug(
+                        "[Discovery] No helm data available",
+                        exc_info=True,
+                    )
+                    break
+
+                for secret_item in secrets.items:
+                    if not secret_item.data or not secret_item.data.get("release", None):
```

Comment:
```secret_item.get``` doesnt exists since secret_item is a class.

But this should work fine: ```release_data = secret_item.data.get("release", None)```